vendor/engines/saas/app/jobs/saas/trials_create_job.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
+  class TrialsCreateJob < ActiveJob::Base
+
+    def perform(params)
+      user = params['current_user']['login']
+      params['url'] = "/settings/#{user}/trialed"
+      Analytics::PageJob.perform_later(params)
+    end
```

Comment:
> It's /settings/{login}/trialed for both, I was under the assumption we want to track the subscriber/owner

We can identify the subscriber/owner from the actual payment information. In GA I _think_ it's more useful to know what user/org gained or lost access.

This would also align with `/settings/{org}/trial`, correct?